docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,278 @@
+# Getting Started With SOCI Snapshotter
+This document is a guide to walk through an example of using the
+soci-snapshotter.  To get everything running we will need to do a few things :
+- prerequisites
+- start a local references-enabled OCI registry
+- push an image to our registry using ctr
+- create and push a SOCI index to our registry
+- run a lazily loaded container from the registry
+
+Along the way we will make sure to inspect each of our actions to ensure that
+they worked.  This is valuable to help learn about the tool and debugging.
+
+## Prerequisites
+**docker** - to confirm that you have docker working please check with `docker
+version` and `docker images` - both should succeed.
+
+**containerd** - to confirm that you have containerd working please check with
+`ctr version` and `ctr i ls`
+
+**ctr** - to confirm that you have ctr, the containerd CLI installed, check with
+`ctr version` and `ctr i ls`
+
+**soci-snapshotter** - this you will need to make sure you can build from
+source. You should be able to run `make` in the soci-snapshotter dir.
+Some common issues :
+- getting the github repository `git clone
+https://github.com/awslabs/soci-snapshotter.git`
+- having the zlib developer tools (zlib is used for decompression).
+Often fixed with `sudo yum install zlib-devel`
+- having the proper go version installed, as of this document please use 1.18+
+
+## Start a local ORAS registry
+A standard OCI registry currently does not support "referrers" feature support.
```

Comment:
double use of the word "support" here; simplify to "Most OCI registries do not currently support the "referrers" feature."